Upgrading.md

```diff
@@ -118,6 +119,15 @@ Follow one of the [installation guides](https://github.com/kata-containers/docum
 $ sudo docker run -ti busybox sh
 ```
 
+# Upgrade from runV
+
+runV and Kata Containers can run together on the same system without affecting each other, as long as they are
+not configured to use the same container root storage, which runV defaults to `/run/runv` and Kata Containers
```

Comment:
Lines 125-126 suggested rewrite (to break up the sentence): not configured to use the same container root storage. Currently, runV defaults to `/run/runv` and Kata Containers defaults to `/var/run/kata-containers`.